.env.example

```
@@ -0,0 +1,111 @@
+# Production Environment Variables - متغيرات بيئة الإنتاج
+# Real production configuration - إعدادات إنتاج حقيقية
+
+# Application Settings - إعدادات التطبيق
+APP_NAME=Super Blue Badge App
```

dotenv-linter (3.3.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)